src/index.js

```diff
@@ -1,6 +1,13 @@
 const js_hamburger = document.getElementById("js-hamburger");
 const js_globalNav = document.getElementById("js-global-nav");
 
+js_hamburger.addEventListener("mouseover",function() {
+  js_globalNav.style.willChange = "transform";
+});
+js_hamburger.addEventListener("mouseout",function() {
+  js_globalNav.style.willChange = "";
+});
+
```

Comment:
I think that this `js _` prefix is no need. because If you will be add it, all variables that exist in js file have to add it. Now, those variables clearly used to defined variables in js.

Reply:
```suggestion
hamburger.addEventListener("mouseover",function() {
  globalNav.style.willChange = "transform";
});
hamburger.addEventListener("mouseout",function() {
  globalNav.style.willChange = "";
});

```

Reply:
**> I think that this `js _` prefix is no need. because If you will be add it, all variables that exist in js file have to add it. Now, those variables clearly used to defined variables in js.**

Thank you.
Fixed.